src/modules/chat_deleted_messages/index.js

```diff
@@ -40,6 +36,12 @@ function onClearChat(name, tags) {
                     $unlinked.text($link.attr('href'));
                     $link.replaceWith($unlinked);
                 });
+
+                if (!settings.get('showDeletedMessages')) {
+                    const $messageClone = $message.clone();
+                    $message.click(() => $message.replaceWith($messageClone));
+                    $message.text('<message deleted>');
```

Comment:
need to make this linkified .. i.e. on hover underline .. greyed out text by enabling our deleted class